build.gradle

```diff
@@ -390,12 +390,12 @@ project(':iceberg-pig') {
 project(':iceberg-spark-runtime') {
   apply plugin: 'com.github.johnrengelman.shadow'
 
-  tasks.build.dependsOn tasks.shadowJar
+  tasks.assemble.dependsOn tasks.shadowJar
```

Comment:
Is this more specific than build? The intent is to only produce the shadow jar, not any other Jar. If `assemble` is a better way to do that and cover more possible tasks users call, great.

Reply:
yes! Checkout build and assemble – https://docs.gradle.org/current/userguide/java_plugin.html#lifecycle_tasks

Reply:
Great, and thanks for the reference!